torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -34,7 +34,7 @@
     rocm_path = subprocess.check_output(["hipconfig", "--rocmpath"]).decode("utf-8")
 except subprocess.CalledProcessError:
     print(f"Warning: hipconfig --rocmpath failed, assuming {rocm_path}")
-except FileNotFoundError:
+except (FileNotFoundError, PermissionError):
```

Comment:
@Gamrix How this can change anything? `PermissionError` was already handled with `pass` in the next lines.

Reply:
Ugh, I think multiple people were fixing this at the same time. That line below was not there when I originally hit the error :/